instrumentation/khttp-0.1/src/main/java/io/opentelemetry/auto/instrumentation/khttp/KHttpDecorator.java

```diff
@@ -42,4 +42,10 @@ protected URI url(RequestWrapper requestWrapper) throws URISyntaxException {
   protected Integer status(Response response) {
     return response.getStatusCode();
   }
+
+  @Override
+  protected String userAgent(RequestWrapper requestWrapper) {
+    // TODO: Find out how to get user agent.
```

Comment:
File an issue about that?

Reply:
Thanks for the diligence! Filed https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/557